src/Syntax/CST/Kinds.hs

```diff
@@ -33,7 +32,10 @@ data EvaluationOrder = CBV | CBN
 -- | A MonoKind is a kind which classifies inhabitated types.
 data MonoKind
   = CBox EvaluationOrder  -- ^ Boxed CBV/CBN
-  | CRep PrimitiveType    -- ^ Primitive type representation
+  | I64Rep
+  | F64Rep
+  | CharRep
+  | StringRep
   deriving (Show, Eq, Ord)
```

Comment:
I don't really understand this change.
The `PrimitiveType` was moved to the type automata instead.
If we don't get rid of this type altogether, this change only makes things more confusing (and more annoying to use in the kind inference later on).